test/e2e/pods.go

```diff
@@ -30,10 +30,59 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+func runLivenessTest(c *client.Client, podDescr *api.Pod) {
+	defer GinkgoRecover()
+	ns := "e2e-test-" + string(util.NewUUID())
+
+	By(fmt.Sprintf("Creating pod %s in namespace %s", podDescr.Name, ns))
+	_, err := c.Pods(ns).Create(podDescr)
+	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("creating pod %s", podDescr.Name))
+
+	// At the end of the test, clean up by removing the pod.
+	defer func() {
```

Comment:
The only issue is that we are ignoring the return value of the Delete function if we do it this way. It might be nicer to have a wrapper func (deletePodOrDie) that checks the err value and calls Expect(err).NotTo(HaveOccurred()) on it.